frontend/src/uxbox/main/ui/keyboard.cljs

```diff
@@ -5,6 +5,10 @@
   (fn [e]
     (= (.-keyCode e) keycode)))
 
+(defn ^boolean alt?
+  [event]
+  (.-altKey event))
+
```

Comment:
I added this in one of the tries, currently not needed but think does not harm leaving it here.